src/views/day/single-event.php

```diff
@@ -58,6 +47,18 @@
 	<?php endif; ?>
 
 </div><!-- .tribe-events-event-meta -->
+
+<!-- Event Cost -->
```

Comment:
You'll need to add a `@version 4.4.whatever` line in this file's docblock at the top of the file